packages/@orchidui-vue/src/DataDisplay/Overview/OcOverview.stories.js

```diff
@@ -5,6 +5,12 @@ export default {
   tags: ["autodocs"],
 };
 export const overviewItem = {
+  argTypes: {
+    percentType: {
+      control: "select",
+      options: ["up", "down"],
```

Comment:
Is this a requirement? If not, up/down should be best to base on percent value (ie. +1% or -1%)

Reply:
No, it isn't. Thanks, I will change